weatherman.py

```diff
@@ -0,0 +1,47 @@
+from Report import Report
+from Reader import Reader
+from Parser import Parser
+import argparse
+import os
+
+
+def directory_path(path):
```

Comment:
As mentioned before, function name should be verb.
This function is actually validating a path but name is not revealing that intent.